bond_token.py

```diff
@@ -0,0 +1,38 @@
+from google.appengine.ext import ndb
+import json
+
+
+class BondToken(ndb.Model):
+    token_dict_str = ndb.TextProperty()
+
+    REQUIRED_KEYS = ["access_token", "refresh_token", "token_type"]
```

Comment:
We don't need to store access_token, it is only good for an hour. 

What is token_type and why do we need to store it?

I think we should also store the date it was created and the user name within the other system.